packages/express/src/middleware/index.ts

```diff
@@ -45,7 +52,7 @@ export function expressMiddleware(appsignal: Client): RequestHandler {
           span.setName(`${method} ${baseUrl}${req.route.path}`)
         }
 
-        span.setSampleData("params", { ...params, ...query })
+        span.setSampleData("params", { ...params, ...query, ...objectBody })
```

Comment:
Not sure this is the best workaround. My suggestion was pretty much limited not considering edge cases.

Just to give you an example, what if params or query or body have a key with the same name? In that case they will be overriden.
Maybe body should be elsewhere?

Reply:
Hi @matiasgarcia, thanks for your feedback.

That would be unlikely, and an odd API design, but it's correct HTTP, and a POST request with query params and a body at the same time could happen.

I added the key `body` when the object is present as you suggested.